### DISCUSSION TITLE
Use a different camera than gameplaycamera

### DISCUSSION THREAD

**Discussion by xAstroBoy** [winstxnhdw/lc-hax#417] [Ideas]
Most enemies are programmed to interact with The gameplay camera, which ControlCompany as well won't utilize, we can make a camera spawner class and overlap any code that uses the camera and detect if we are using the custom camera or not.

That would as well fix Coilhead not moving when we are looking at him under possession because we are technically watching him.

**Comment by winstxnhdw** [comment]
I think @LTYGUY tried doing that the first time but the screen doesn't update on the new camera. You should try experimenting with it first.
